ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -82,16 +82,6 @@ object Cli {
       .text("TLS: The crt file to be used as the the trusted root CA. Applied to all endpoints.")
       .action(cacrtConfig)
 
-    opt[Double](name = "command-submission-ttl-scale-factor")
```

Comment:
Maybe we want to keep it in and just let it do nothing?

Reply:
Why keeping an option that doesn't do anything? To not break external test setups?

Reply:
Yes, that would've been the idea. But we can add it back if it turns out to be a problem, which I don't think it will be.